include/MidiPort.h

```diff
@@ -111,6 +115,40 @@ class MidiPort : public Model, public SerializingObject
 		return outputChannel() ? outputChannel() - 1 : 0;
 	}
 
+	//! Returns the current number of channels in the Lower MPE Zone, or 0 if there are no member channels or MPE is disabled
+	int MPELowerZoneChannels() const
+	{
+		return MPEEnabled() && m_MPELowerZoneChannelsModel.value() > 1
+			? m_MPELowerZoneChannelsModel.value()
+			: 0;
+	}
+	//! Returns the current number of channels in the Upper MPE Zone, or 0 if there are no member channels or MPE is disabled
+	int MPEUpperZoneChannels() const
+	{
+		return MPEEnabled() && m_MPEUpperZoneChannelsModel.value() > 1
+			? m_MPEUpperZoneChannelsModel.value()
+			: 0;
+	}
+
+	// Returns the current active MPE zone (Lower or Upper)
+	MPEManager::MPEZone MPEActiveZone() const
+	{
+		switch (m_MPEZoneModel.value())
+		{
+			case 0:
+				return MPEManager::MPEZone::Lower;
+			case 1:
+				return MPEManager::MPEZone::Upper;
+			default:
+				return MPEManager::MPEZone::Lower;
+		}
+	}
+
+	MPEManager* mpeManager()
+	{
+		return &m_mpeManager;
+	}
```

Comment:
```suggestion
	MPEManager& mpeManager()
	{
		return m_mpeManager;
	}
```
Could return a reference since it's never null.